framework/src/org/checkerframework/common/value/qual/MinLenFieldInvariant.java

```diff
@@ -1,4 +1,4 @@
-package org.checkerframework.checker.index.qual;
+package org.checkerframework.common.value.qual;
```

Comment:
I updated the Javadoc.  For now, let's just leave it as is.  After this PR is merged, we can decide if we want to change it.